server/views/partials/summary-card-with-steps.njk

```diff
@@ -0,0 +1,60 @@
+{% from "../components/summary-card/summary-card.njk" import summaryCard %}
+
+{% macro summaryListWithSteps (goal) %}
+    {% call summaryCard({
+        title: {
+            text:  goal.title
+        },
+        actions: {
+          items: [
+            {
+              href: "/edit-goal",
+              text: "Change goal",
+              visuallyHiddenText: "Change goal"
+            },
+            {
+              href: "/add-change-steps",
+              text: "Add or change steps",
+              visuallyHiddenText: "Add or change steps"
+            },
+            {
+              href: "/remove-goal",
+              text: "Remove goal",
+              visuallyHiddenText: "Remove goal"
+            }
+          ]
+        }
+    }) %}
+
+    <p class="govuk-body">Aim to achieve {{ goal.targetDate | formatSimpleDate }}</p>
+    {% if goal.steps and goal.steps.length > 0 %}
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Step</th>
+        <th scope="col" class="govuk-table__header">Who will do this</th>
+        <th scope="col" class="govuk-table__header">Due date</th>
+      </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+      {% for step in goal.steps %}
+      <tr class="govuk-table__row">
+        <td class="govuk-table__cell">{{ step.description }}</td>
+        <td class="govuk-table__cell">{{ step.actors | getActors | join(', ') }}</td>
+        <td class="govuk-table__cell">{{ step.creationDate | formatSimpleDate }}</td>
+      </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+    {% else %}
+      <strong class="govuk-body govuk-!-font-weight-bold">No steps added</strong>
+      <hr></hr>
+    {% endif %}
+    <p class="govuk-body-s">
+        Area of need: {{ goal.areaOfNeed.name }}<br>
+        {% if goal.relatedAreasOfNeed.length > 0 %}
+          Also relates to: {{ goal.relatedAreasOfNeed | getRelatedAreaOfNeed | join(', ') }}
+        {% endif %}
+    </p>
```

Comment:
Should the buttons for moving goals up and down be here?

Reply:
Dont think its part of this story yet.

Reply:
added